lib/bolt/node/winrm.rb

```diff
@@ -27,12 +27,20 @@ def connect
         transport = :ssl
       end
       endpoint = "#{scheme}://#{host}:#{port}/wsman"
+
+      if @kerberos
```

Comment:
The result of making `kerberos` separate from `insecure`, with a default scheme of `http`, means you usually need to specify `--insecure` with `--kerberos`. We could change to kerberos implying HTTP... there's a chance that the kerberos "transport" can support HTTPS, but I haven't looked into the library enough to see if it'll do that.